ckanext/unaids/theme/templates/home/layout1.html

```diff
@@ -1,10 +1,10 @@
 <div role="main" class="hero">
   <div class="container">
     <div class="row row1">
-      <div class="col-md-6 col1">
+      <div class="col-md-6 col1 hidden-xs">
```

Comment:
Hiding this div above `promoted` block looks a little off. Have you compared the layout1 template we're extending with the original core CKAN and potentially see how it's resolved there?

Reply:
I think using hidden-xs is not a bad solution, but I just couple of points for consideration:

1. Have you considered whether it is easy to actually show the content in mobile view? It's worth giving maybe 10 minutes thought to - fiddling in the browsers inspector console - if after 10 minutes you don't have an easy fix just don't bother. 
2. We need to ensure that the screen size at which the div content disappears exactly aligns with the screen size at which the div itself disappears.  The best way to do this would be to not hide the content at all, but just have one place where the entire div is hidden.  i.e. check which bit of the CSS is already doing the hiding, and ensure that the entire div is hidden with a single CSS rule.

Reply:
@jonathansberry Yea I think we can keep it on mobile. see:
- https://fjelltopp.atlassian.net/browse/ADX-812
- https://github.com/fjelltopp/ckanext-unaids/pull/195